auxiliary_files/config_files/mumble-server.service.in

```diff
@@ -0,0 +1,36 @@
+[Unit]
+Description=Mumble server
+After=network.target
+Wants=network-online.target
+
+[Service]
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+CapabilityBoundingSet=CAP_NET_BIND_SERVICE
+ExecStart=@MUMBLE_INSTALL_ABS_EXECUTABLEDIR@/@MUMBLE_SERVER_BINARY_NAME@ -ini @MUMBLE_INSTALL_ABS_SYSCONFDIR@/mumble-server.ini -fg
```

Comment:
What does `@MUMBLE_INSTALL_ABS_EXECUTABLEDIR@` resolve to?
In this case it should not include/consider `DESTDIR`, in case that is happening here.

Reply:
CMAKE_INSTALL_PREFIX/bin (as an absolute path)

What DESTDIR? 👀

Reply:
Ah, excellent. I think CMAKE_INSTALL_PREFIX is automatically prefixed with DESTDIR, so everything should work as intended.

```
DESTDIR="$pkgdir" make install -C build
```
or

```
DESTDIR="$pkgdir" cmake --install build
```

Is used to install all files (adhering to the given prefix, etc.) into a given directory, instead of directly to the root filesystem (`/`). This is commonly used in packaging, as all files are installed into a directory, which is then turned into a package and can be installed by a package manager to the target system (only containing the files of that package), allowing the package manager system to keep track of all the contained files.

Reply:
Eh, wait, I mean, not excellent. The DESTDIR should not end up in this of course.
I guess it would be awesome to test this before merging! I'll try to build this tomorrow and give more feedback!

Reply:
Hm, CMAKE_INSTALL_PREFIX is resolved at configure time, so I don't see how an env variable during make install could influence this  :thinking:

Testing sounds good though :+1: